boards/arm/nucleo_f091rc/nucleo_f091rc_defconfig

```diff
@@ -17,11 +18,18 @@ CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 
+# Pinctrl driver
+CONFIG_PINCTRL=y
+CONFIG_PINCTRL_PINMUX=y
+CONFIG_PINCTRL_STM32=y
+
 # Pinmux Driver
-CONFIG_PINMUX=y
+# replaced by Pinctrl driver
 
 # GPIO Controller
 CONFIG_GPIO=y
+CONFIG_GPIO_PINCTRL=y
+CONFIG_GPIO_STM32=y
```

Comment:
Move to arch/arm/soc/st/stm32/<stm32f0xx or common>/

Reply:
@erwango Not quite shure what to do here?